### PR TITLE
Update mining-token-rewards.md

### PR DESCRIPTION
**Pull request by rawrmaan**
Fixed typos, made some things clearer.

### CODE REVIEW

File: blockchain/mining-token-rewards.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Every [epoch](mining-token-rewards.md), the current consensus group mines some a
 | Network Data Transfer | Distributed each epoch to Hotspots that route LongFi sensor data for sensors on the Network during that epoch.  |
 
 {% hint style="info" %}
-**Do I Have To Actively Participate to Earn Rewards Once My Hotspot is Deployed?**
+**Do I Have To Actively Participate to Earn Rewards Once My Hotspot is deployed?**
```
Comment (jthiller) [inline blockchain/mining-token-rewards.md:29]: It's odd that this is all title case except for the last word.

Comment (rawrmaan) [reply blockchain/mining-token-rewards.md:29]: Oops. Good catch. I was sort of removing title case from things but then decided to revert it to keep things consistent with the rest of the docs. Missed one word ;)

File: blockchain/mining-token-rewards.md [hunk 2]
```diff
@@ -170,7 +170,7 @@ Looking to get advice from 1000s of community members on how to optimize your HN
...
+* **Ensuring you aren't the only Hotspot in your area** is the method most likely to increase your earnings. If you are in an area with three or more Hotspots you are likely to participate as a `PoC Challengee` and `Witness` more PoC challenges that are happening around you. These are the two highest `HNT` distributions for Proof of Coverage per epoch, so optimizing around them has a high impact.
+* **Upgrading to a larger antenna** will help in situations where you have other Hotspots nearby but either fail challenges that you participate in or do not witness challenges that they are participating in. Note: Upgrading to a larger antenna will not help in situations when you are the only Hotspot in the area, as your only HNT earnings will come from issuing challenges which only uses your internet connection. Figuring out which antenna to upgrade to is a complex question, and we recommend joining our [Community Slack](http://chat.helium.com) where plenty of antenna discussion occurs.
```
Comment (jthiller) [inline blockchain/mining-token-rewards.md:174]: I'll just call it out here instead of a new PR - but this should point to Discord instead of Slack.

Comment (rawrmaan) [reply blockchain/mining-token-rewards.md]: Fixed.

**Comment by PharkMillups** [comment]
Looks good. Thanks, Arman. Merging.

**Pull request merged**